packages/opentelemetry-node/test/instrumentation/PluginLoader.test.ts

```diff
@@ -137,7 +136,7 @@ describe('PluginLoader', () => {
 
   describe('.load()', () => {
     afterEach(() => {
-      delete process.env[ENV_PLUGIN_DISABLED_LIST];
+      delete process.env['OTEL_NO_PATCH_MODULES'];
```

Comment:
why did you remove here predefined string in favour of using not predefined, we usually do opposite - define a constant and then reuse it.

Reply:
the const disappeared, because in the actual codebase it's referred to as a field, in tests literal values are usually fine, but let me know what you'd like to see here